### ISSUE TITLE
Setup appveyor builds of binaries

### ISSUE THREAD

**Issue by KOLANICH** [twogood/unshield#90]
Appveyor allows automatically building binaries after every commit and publishing artifacts.

**Comment by twogood** [comment]
Thanks for your suggestion! Where would I put these artifacts?

**Comment by KOLANICH** [comment]
Appveyor stores them itself (you only need to specify in appveyor.yaml what to store) and allows links to them, if their name doesn't change from build to build.

**Comment by twogood** [comment]
Maybe it is possible to create an .msi package like this?
http://dominoc925.blogspot.com/2016/09/create-windows-installer-using-cmake.html

**Comment by KOLANICH** [comment]
>It seems like there are three types of artefacts allowed:

Any types of artifacts are allowed. How to setup build you can see here: https://github.com/jpype-project/jpype/blob/devel/appveyor.yml

(not a very good example, since some artifacts are uploaded manually rather than using `artifacts` dict in appveyor.yml, but the build matrix should be useful)

**Comment by twogood** [comment]
Seems I could specify the .exe as an artifact!
https://ci.appveyor.com/project/twogood/unshield/build/artifacts

**Comment by twogood** [comment]
Maybe fixed by #105

**Comment by KOLANICH** [comment]
>The build job does not contain any artifacts.

**Comment by twogood** [comment]
> > The build job does not contain any artifacts.

Yet! :)

**Comment by TheComputerGuy96** [comment]
I see this project has switched to GitHub Actions which does provide artifacts after logging in to GitHub (which means the issue is effectively solved)

GitHub releases don't have any artifacts though (which makes things harder for Windows users) so there's still some work to do

**Comment by twogood** [comment]
Yeah, what is missing is putting the artifacts somwhere permanent...
